ui/components/ui/currency-display/currency-display.component.js

```diff
@@ -3,6 +3,13 @@ import PropTypes from 'prop-types';
 import classnames from 'classnames';
 import { useCurrencyDisplay } from '../../../hooks/useCurrencyDisplay';
 import { EtherDenomination } from '../../../../shared/constants/common';
+import Box from '../box/box';
+import { Text } from '../../component-library';
+import {
+  AlignItems,
+  DISPLAY,
+  FLEX_WRAP,
```

Comment:
Can we update these deprecated constants to the new enums? Will need to update instances to use PascalCase
```suggestion
  Display,
  FlexWrap,
```